pkg/hubble/filters/ip_test.go

```diff
@@ -157,6 +157,62 @@ func TestIPFilter(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "source cidr",
+			args: args{
+				f: []*flowpb.FlowFilter{{SourceIp: []string{"1.1.1.0/24", "f00d::/16"}}},
+				ev: []*v1.Event{
+					{Event: &flowpb.Flow{IP: &flowpb.IP{Source: "1.1.1.1", Destination: "2.2.2.2"}}},
+					{Event: &flowpb.Flow{IP: &flowpb.IP{Source: "2.2.2.2", Destination: "1.1.1.1"}}},
+					{Event: &flowpb.Flow{IP: &flowpb.IP{Source: "f00d::a10:0:0:9195", Destination: "ff02::1:ff00:b3e5"}}},
+					{Event: &flowpb.Flow{IP: &flowpb.IP{Source: "ff02::1:ff00:b3e5", Destination: "f00d::a10:0:0:9195"}}},
+				},
+			},
+			want: []bool{
+				true,
+				false,
+				true,
```

Comment:
I'm not super familiar with these tests, but is this saying that `f00d::a10:0:0:9195` matches `1.1.1.0/24`?

Reply:
no. so the filter is:
```
f: []*flowpb.FlowFilter{{SourceIp: []string{"1.1.1.0/24", "f00d::/16"}}}
```
this filter returns true if the source ip (in this case `f00d::a10:0:0:9195`) matches either `"1.1.1.0/24"` or `"f00d::/16"`.